nova/virt/vmwareapi/cluster_util.py

```diff
@@ -61,12 +58,39 @@ def _create_host_group_spec(client_factory, name, host_refs, operation="add",
     else:
         group.host = host_refs
 
+    return group
+
+
+def create_group_spec(client_factory, group, operation):
+    """Create a ClusterGroupSpec object"""
+    if operation not in ('add', 'edit', 'remove'):
+        msg = 'Invalid operation for ClusterGroupSpec: {}'.format(operation)
+        raise exception.ValidationError(msg)
```

Comment:
Wouldn't the standard-lib exception [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) be more appropriate here?

Reply:
Maybe. `ValidationError` is a `NovaException`, which can be handled differently (e.g. by Nova). Additionally, other functions inside this module use `ValidationError` (or previously used `Invalid()` instead, which is also a `NovaException`) already, so for the sake of consistency, I would keep it that way.